app/extensions/brave/locales/es/menu.properties

```diff
@@ -87,7 +87,6 @@ preferences=Preferencias…
 settings=Ajustes…
 bookmarksManager=Administrador de marcadores…
 importBrowserData=Importar datos del navegador...
-reportAnIssue=Reportar un problema
```

Comment:
The non-English (ex: `en-US`) files can be left alone; they'll get stripped out when the next internationalization pull happens :smile: